airsonic-main/src/main/java/org/airsonic/player/service/VersionService.java

```diff
@@ -243,6 +244,11 @@ private void readLatestVersion() throws IOException {
         try (CloseableHttpClient client = HttpClients.createDefault()) {
             ResponseHandler<String> responseHandler = new BasicResponseHandler();
             content = client.execute(method, responseHandler);
+        } catch (ConnectTimeoutException e) {
+          LOG.warn("Got a timeout when trying to reach {}", VERSION_URL);
+          latestFinalVersion = getLocalVersion();
+          latestBetaVersion = getLocalVersion();
```

Comment:
We should not set these two variables. Leaving them unset should make the UI not display a new version being available.